srv/salt/ceph/packages/common/default.sls

```diff
@@ -1,11 +1,28 @@
+{% os = salt['grains.get']('os') %}
```

Comment:
Also, we should probably do this as `os_family`, rather than `os`, because `os_family` is `Debian` for both Debian and Ubuntu, which'll save an if later.